### PR TITLE
Fix `modernize-use-emplace` and `performance-unnecessary-copy-initialization` reported by clang-tidy

### PR DESCRIPTION
**Pull request by PragmaTwice**

### CODE REVIEW

**Review by torwig — APPROVED** [review]
LGTM

**Review by git-hulk — APPROVED** [review]

File: src/storage/storage.cc [hunk 1]
```diff
@@ -302,12 +302,12 @@ Status Storage::Open(bool read_only) {
 
   std::vector<rocksdb::ColumnFamilyDescriptor> column_families;
   // Caution: don't change the order of column family, or the handle will be mismatched
-  column_families.emplace_back(rocksdb::ColumnFamilyDescriptor(rocksdb::kDefaultColumnFamilyName, subkey_opts));
-  column_families.emplace_back(rocksdb::ColumnFamilyDescriptor(kMetadataColumnFamilyName, metadata_opts));
-  column_families.emplace_back(rocksdb::ColumnFamilyDescriptor(kZSetScoreColumnFamilyName, subkey_opts));
-  column_families.emplace_back(rocksdb::ColumnFamilyDescriptor(kPubSubColumnFamilyName, pubsub_opts));
-  column_families.emplace_back(rocksdb::ColumnFamilyDescriptor(kPropagateColumnFamilyName, propagate_opts));
-  column_families.emplace_back(rocksdb::ColumnFamilyDescriptor(kStreamColumnFamilyName, subkey_opts));
+  column_families.emplace_back(rocksdb::kDefaultColumnFamilyName, subkey_opts);
```
Comment (tisonkun) [inline src/storage/storage.cc:305]: Inconsistent. Why not barely `kDefaultColumnFamilyName`?

**Review by tisonkun — CHANGES_REQUESTED** [review]

File: src/storage/storage.cc (hunk 1 above)
Comment (PragmaTwice) [reply src/storage/storage.cc:305]: It is also `rocksdb::kDefaultColumnFamilyName` in the original code. I did not change it.

Comment (tisonkun) [reply src/storage/storage.cc:305]: Oops. Yes this constant comes from another namespace :)

**Review by tisonkun — APPROVED** [review]

**Pull request merged**